index.html

```diff
@@ -8,6 +8,122 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="price">
+        <h2 class="visually-hidden">Наши тарифы</h2>
+        <table class="table">
+          <thead>
+            <tr class="table__row">
+              <th class="table__header"></th>
+              <th class="table__header">
+                <h2 class="table__plan-name">База</h2>
+                <p class="table__plan-price">1,99 USD</p>
+              </th>
+              <th class="table__header table__header_hit">
+                <h2 class="table__plan-name">Стандарт</h2>
+                <p class="table__plan-price">3,99 USD</p>
+              </th>
+              <th class="table__header">
+                <h2 class="table__plan-name">Анлим</h2>
+                <p class="table__plan-price">9,99 USD</p>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="table__row">
+              <td class="table__cell table__cell_content_name">
+               <p class="table__feature">Розовый фильтр</p>
+              </td>
+              <td class="table__cell">
+                <div class="table__cell-content">
```

Comment:
Do you really need an extra `div` container in the table cell or the style can be handled in any other way?

Reply:
@5hraddha I reckon I can do away with the div `table__cell-content` and transfer its CSS properties to the div `table__cell`.Am i right?